crates/re_types/definitions/rerun/blueprint/components/included_content.fbs

```diff
@@ -11,7 +11,7 @@ namespace rerun.blueprint.components;
 
 
 /// All the contents in the container.
-table IncludedContents (
+table IncludedContent (
```

Comment:
Surprised this isn't arrow transparent

Reply:
Components are always arrow transparent by default.